awkward-cpp/CMakeLists.txt

```diff
@@ -50,7 +49,12 @@ file(GLOB_RECURSE LIBAWKWARD_SOURCES CONFIGURE_DEPENDS "src/libawkward/*.cpp")
 add_library(awkward-parent INTERFACE)
 target_compile_definitions(awkward-parent INTERFACE VERSION_INFO="${SKBUILD_PROJECT_VERSION}")
 target_include_directories(awkward-parent INTERFACE include)
-target_compile_features(awkward-parent INTERFACE cxx_std_11)
+target_compile_features(awkward-parent INTERFACE cxx_std_17)
+set_target_properties(
+  awkward-parent
+  PROPERTIES CXX_VISIBILITY_PRESET hidden
+             VISIBILITY_INLINES_HIDDEN ON
+             CXX_EXTENSIONS NO)
```

Comment:
This is an interface target? Why is CMake letting you set these non-interface properties on an interface target? There's no built component to apply them to, and they are not inheritable last I checked.

Reply:
(Interface properties start with `INTERFACE_`)

Reply:
Oops, these should also be removed. Let me do another pass.